test/LieTheory/WeylGroup.jl

```diff
@@ -203,30 +207,40 @@
     w0 = longest_element(W)
     @test is_in_normal_form(w0)
     @test word(w0) == UInt8[1, 2, 1]
+    @test letters(w0) == UInt8[1, 2, 1]
+    @test syllables(w0) == [1 => 1, 2 => 1, 1 => 1]
```

Comment:
```suggestion
    a2_w0_word = UInt8[1, 2, 1]
    @test word(w0) == a2_w0_word
    @test letters(w0) == Int.(a2_w0_word)
    @test syllables(w0) == [Int(i) => 1 for i in a2_w0_word]
```
and then similar for the other cases below